components/Init/src/hpet.c

```diff
@@ -0,0 +1,739 @@
+/*
+ *  High Precision Event Timer emulation
+ *
+ *  Copyright (c) 2007 Alexander Graf
+ *  Copyright (c) 2008 IBM Corporation
+ *
+ *  Authors: Beth Kon <bkon@us.ibm.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, see <http://www.gnu.org/licenses/>.
+ *
+ * *****************************************************************
+ *
+ * This driver attempts to emulate an HPET device in software.
+ */
+
+/*
+ * Copyright 2019, DornerWorks
+ *
+ * SPDX-License-Identifier: GPL-2.0-only
```

Comment:
This should say `LGPL-2.0-or-later`

Reply:
(You''l then have to add the corresponding license text to `LICENSES/` for the license check to succeed -- you can use the attached [LGPL-2.0-or-later.txt](https://github.com/seL4/camkes-vm/files/13357934/LGPL-2.0-or-later.txt))

Reply:
Done!